weheat/abstractions/heat_pump.py

```diff
@@ -21,8 +26,10 @@ def __init__(self, api_url: str, uuid: str) -> None:
         self._api_url = api_url
         self._uuid = uuid
         self._last_log = None
+        self._energy_consumption = None
 
     def get_status(self, access_token: str):
+        """Updates the HEatPump instance with data from the API."""
         try:
```

Comment:
```suggestion
    def get_status(self, access_token: str):
        """Updates the heat pump instance with data from the API."""
        try:
```

typo + lower case "heat pump" for consistency.